rc/lsp.kak

```diff
@@ -342,45 +342,32 @@ have_kakoune_feature_filtertext = ${kak_opt_lsp_have_kakoune_feature_filtertext}
 " | eval "${kak_opt_lsp_cmd} --request") > /dev/null 2>&1 < /dev/null & }
 }}
 
-declare-option -hidden str lsp_completion_selection ""
+declare-option -hidden str lsp_completion_inserted_range
 
-define-command -hidden lsp-completion-dismissed -docstring "Called when the completion pager is closed" %{
-    set-option window lsp_completion_selection %val{hook_param}
-    try %{
-        evaluate-commands -draft %{
-            select %opt{lsp_completion_selection}
-        }
-    } catch %{
-        # Completion was cancelled, nothing was inserted
```

Comment:
If I understand correctly, you moved the logic in the regex to trigger the hook with a `.+` to ensure the completion is accepted.

It could be worth to readd comments to explain it.

Reply:
Yep. An empty hook parameter means no competion candidate was inserted.
The documentation of InsertCompletionHide does not state this explicitly, we should probably improve that.
Added a comment.